### PR TITLE
Milestone 8: Portfolio - details popup window

### PR DESCRIPTION
**Pull request by dqarias**
In this milestone we: 

- Implement the following interactions:
    - [ ] When the user clicks (or taps) the button to check project details, the popup with details about the project appears.
    - [ ] When the user clicks (or taps) the close (X) button, the popup disappears.
- Implement popups for both mobile and desktop screen sizes.
- Associate each project with the popup window details using a JavaScript array to store all of the information for all projects.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -74,148 +74,9 @@ <h1 class="hero__title">
...
-              <li>javaScript</li>
-            </ul>
-            <div class="btn-card">
-              <button class="btn btn-primary" type="button">See Project</button>
-            </div>
-          </div>
-        </div>
+      <section id="works" class="works"> 
       </section>
+
```
Comment (danmainah) [inline index.html:79]: - [ ] @Team great work here, however, there is a slight error in your desktop design, kindy make sure the edges of your paragraph and buttons align with the image edges like in the Figma design:

Figma Design          |  Your design     
:-------------------------:|:-------------------------:
image |  image

**Review by danmainah — CHANGES_REQUESTED** [review]
Hi @dqarias and @kiboma2021 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✔️ No linter errors
✔️ Github flow was correctly used
✔️  Good PR and Description


# Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: index.html (hunk 1 above)
Comment (dqarias) [reply index.html:79]: I've Just done it. Thank you for your feedback

**Review by gtekle — APPROVED** [review]
Hi @dqarias :wave: , 

# STATUS: APPROVED :clap: :clap: :clap: 🎉 🎉 🎉 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
## Congratulations!  🎉 
mazel tov


## Highlights
:heavy_check_mark: Clean and organized code
:heavy_check_mark: Nice UI design
:heavy_check_mark: Good use of Github flow
:heavy_check_mark: Meaningful Commit messages

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**